### DISCUSSION TITLE
No internet access in virt-manager VMs (DNS)

### DISCUSSION THREAD

**Discussion by Alex47260** [netblue30/firejail#6546] [Q&A] [answered]
Hi to the Firejail team and community,

Firstly, many thanks for this great solution that I recently started to use on my Linux (Fedora 40) desktop and I find it really cool (I actually mean impressive)!

My question regards Firejail used in conjuction with virt-manager. 

Virt-manager is my preferred option to run VMs and it has been working fine. I have now installed Firejail, which works fine with Firefox or Wireshark so I am really happy with it.

Having tried to run my VMs again today - first time after Firejail install - my VMs don't have internet access. I assumed these might be "confined" somehow although I don't see anything related to virt-manager when running `firejail --top` or f`irejail --list`. I checked `/etc/firewall` and there is no virt-manager.profile yet and seeing post #4526, I understand a custom profile is needed.

There is a good deal of information in post #4526 already but would someone have a virt-manager.profile file and share it please? I'd do it and share it gladly but I am not there yet :-D

 Cheers,

**Comment by rusty-snake** [comment, marked as answer]
Does 096d5a2a2db1b86e3312ef1d15ae8db1d679848c fix it for you?

**Reply by Alex47260** [reply]
Many thanks @rusty-snake for the swift comment! 

Actually, that is not a Firejail issue...sorry...but you helped me and pointed to the right place, i.e. to dnsmasq - which I replaced on my Fedora computer by Unbound - which was not (re)allowed to access the host, hence what reported with the VMs : the VM in virt-manager can now have network access, as before.

Not changing subject :-D but just to understand this is abit better, does virt-manager require a custom profile then (cf. #4526) ? And wondering please what Firejail activity should be seen when virt-manager is used? 

Cheers,

**Reply by rusty-snake** [reply]
Unless you want to sandbox virt-manager (the UI, not libvirt and not the VM) you do not need a firejail profile for it.
